src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -957,5 +968,19 @@ internal static uint FlipBit(uint value, int index)
         {
             return value ^ (1u << index);
         }
+
+        /// <summary>
+        /// Flip the bit at a specific position in a given value.
+        /// Similar in behavior to the x86 instruction BTC (Bit Test and Complement).
+        /// /// </summary>
+        /// <param name="value">The value.</param>
+        /// <param name="index">The zero-based index of the bit to flip.
+        /// Any value outside the range [0..63] is treated as congruent mod 64.</param>
+        /// <returns>The new value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static ulong FlipBit(ulong value, int index)
+        {
+            return value ^ (1UL << index);
```

Comment:
Minor nit: we use `u` above but `UL` here... it'd be nice to be consistent on capitalization.

Reply:
Changed to `ul`